testdata/.tranor/envs.json

```diff
@@ -0,0 +1,14 @@
+[
```

Comment:
I guess you are right. I thought there would be a default configuration file installed without EF stuff, but I don't know if that makes sense.